charts/netbox/values.yaml

```diff
@@ -1324,6 +1324,9 @@ housekeeping:
   ##     memory: 1024Mi
   ##
   resources: {}
+  ## @param housekeeping.readOnlyPersistence Whether to mount media, script, and report directories as read-only
+  ##
+  readOnlyPersistence: false
```

Comment:
Just a thought - should persistence setting be inside the `.Values.persistence|scriptsPersistence` etc.  block?

Reply:
ahh, yeah, duh

Reply:
Oh, actually, I started on this and I realized no; because you want the "main" pod to be read-write for all directories, and the others to be read-only for all directories; so it's not main/scripts/reports as the taxonomy, it's netbox/worker/housekeeping.